pages/glossary/visible.md

```diff
@@ -3,6 +3,8 @@ title: Visible
 key: visible
 unambiguous: true
 objective: true
+input_aspects:
```

Comment:
I'm not sure we can leave "DOM tree" outside. 
For example, with the `<details><summary>` elements, the details become visible when the `<details>` element gains an `open` attribute.